ui/js/graphs.js

```diff
@@ -84,6 +84,12 @@ perf.controller('GraphsCtrl', [
                                      function(resultSetId) {
                                        return (resultSetId < dataPoint.resultSetId);
                                      });
+        var triggerCounter = _.countBy(phSeries.flotSeries.resultSetData,
```

Comment:
Wondering if we might want to use the term 'retrigger' generally where applicable in this PR? If nothing else for UI association purposes with Treeherder. eg. `var retriggerNum =`, or something :)